modules/WordCloudPlugin/src/main/java/net/clementlevallois/wordcloud/TopTermExtractor.java

```diff
@@ -0,0 +1,186 @@
+/*
+ * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
+ * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
+ */
+package net.clementlevallois.wordcloud;
+
+import java.io.IOException;
+import java.io.StringReader;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import net.clementlevallois.stopwords.StopWordsRemover;
+import net.clementlevallois.umigon.model.TextFragment;
+import net.clementlevallois.umigon.model.TypeOfTextFragment.TypeOfTextFragmentEnum;
+import net.clementlevallois.umigon.tokenizer.controller.UmigonTokenizer;
+import net.clementlevallois.utils.Multiset;
+import org.gephi.graph.api.Column;
+import org.gephi.graph.api.Graph;
+import org.gephi.graph.api.GraphModel;
+import org.gephi.graph.api.Node;
+import org.gephi.graph.api.NodeIterable;
+import org.gephi.io.importer.api.Container;
+import org.gephi.io.importer.api.ContainerUnloader;
+import org.gephi.io.importer.api.ImportController;
+import org.gephi.io.importer.plugin.file.ImporterGEXF;
+import org.gephi.io.importer.spi.FileImporter;
+import org.gephi.io.processor.plugin.DefaultProcessor;
+import org.gephi.project.api.ProjectController;
+import org.openide.util.Exceptions;
+import org.openide.util.Lookup;
+
+/**
+ *
+ * @author LEVALLOIS
+ */
+public class TopTermExtractor {
+
+    public static void main(String args[]) throws IOException, Exception {
+        new TopTermExtractor().importFromFile();
+    }
+
+    public void importFromFile() throws IOException, Exception {
```

Comment:
I'm not understanding what is this about? Maybe something you used for testing and forgot to remove?

Reply:
Indeed

Reply:
I suggest to keep it (minus the file name), it helps me maintain it